2-WebApp-graph-user/2-1-Call-MSGraph/README.md

```diff
@@ -277,14 +356,87 @@ HTML table displaying the properties of the *me* object as returned by Microsoft
 </table>
 ```
 
-## Next steps
+## Deployment
+
+### Deploying web app to Azure App Services
+
+There is one web app in this sample. To deploy it to **Azure App Services**, you'll need to:
+
+- create an **Azure App Service**
+- publish the projects to the **App Services**, and
+- update its client(s) to call the website instead of the local environment.
+
+#### Publish your files (WebApp-OpenIDConnect-DotNet-graph-v2)
+
+##### Publish using Visual Studio
+
+Follow the link to [Publish with Visual Studio](https://docs.microsoft.com/visualstudio/deployment/quickstart-deploy-to-azure).
+
+##### Publish using Visual Studio Code
+
+1. Install the VS Code extension [Azure App Service](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice).
+1. Sign-in to App Service using Azure AD Account.
+1. Open the WebApp-OpenIDConnect-DotNet-graph-v2 project folder.
+1. Choose View > Terminal from the main menu.
+1. The terminal opens in the WebApp-OpenIDConnect-DotNet-graph-v2 folder.
+1. Run the following command:
+
+    ```console
+    dotnet publish --configuration Release
+    ```
+
+1. Publish folder is created under path ``bin/Release/<Enter_Framework_FolderName>``.
+1. Right Click on **Publish** folder and select **Deploy to Web App**.
+1. Select **Create New Web App**, enter unique name for the app.
+1. Select Windows as the OS. Press Enter.
+
+#### Disable Azure App Services default authentication (WebApp-OpenIDConnect-DotNet-graph-v2)
+
+1. Go to [Azure portal](https://portal.azure.com), and locate your project there.
+    - On the Settings tab, select **Authentication/Authorization**. Make sure **App Service Authentication** is `off` (as we are using our own **custom** authentication logic). 
+    - Select **Save**.
+1. Browse your website. If you see the default web page of the project, then the publication was successful.
```

Comment:
@derisen , this section can now be dropped from the code gen

Reply:
I'll drop it but leave a note in its place. I think people are tempted to switch it on